compiler/ballerina-parser/src/test/resources/expressions/xml-template/xml_template_source_31.bal

```diff
@@ -0,0 +1,20 @@
+function foo() {
+    xml a = xml `<![CDATA[some text]]>`;
```

Comment:
Can add a test case for `<![CDATA[<greeting>Hello, world!</greeting>]]> `

Reply:
Will add